scenes/game/local_game.gd

```diff
@@ -166,6 +172,12 @@ class Strike:
 		if performing_player == initiator:
 			return initiator_wild_strike
 		return defender_wild_strike
+		
+	func get_player_strike_from_gauge(performing_player : Player) -> bool:
+		if performing_player == defender:
+			return true
```

Comment:
Shouldn't this be false? The defender can't strike from gauge?

Reply:
whoops